### ISSUE TITLE
Unnamed error in wnsm/live_sensor.py:68

### ISSUE THREAD

**Issue by michib-xx** [DarwinsBuddy/WienerNetzeSmartmeter#237]
I have added an account with in total 5 smart meters. 
Only 3 from 5 are actually set as HA sensors (with constant update errors - no values read).
However, main issue is found in the log:

(used 'x's to anonymize)

```
Please file an issue with this error and (anonymized) payload in github {'hasSmartMeter': True, 'isDataDeleted': False, 'zaehlpunkt': 'AT00100000000000000010000xxxxxx', 'type': 'TAGSTROM', 'address': 'xxxxx, 1xxx0 Wien', 'zip': '1xx0'} {'consumptionYesterdayValue': x668, 'consumptionYesterdayValidated': True, 'consumptionYesterdayTimestamp': '2024-06-10T22:00:00.000Z', 'consumptionDayBeforeYesterdayValue': x694, 'consumptionDayBeforeYesterdayValidated': True, 'consumptionDayBeforeYesterdayTimestamp': '2024-06-09T22:00:00.000Z'} {'lastValue': xx19637, 'lastReading': '2024-06-11T22:00:00.000Z', 'lastValidated': True, 'lastType': '1-2:1.8.0'} {'optIn': True, 'consumptionAverage': 0, 'consumptionMinimum': 0, 'consumptionMaximum': 0}
```

What is wrong?
This very Zählpunkt is not added as sensor.

**Comment by tschoerk** [comment]
Sounds like the consumption error mostly fixed in #233.
For multiple zaehlpunkte you will probably run into a lot of other bugs. Until the main application is updated, you can try [this](https://github.com/tschoerk/WienerNetzeSmartmeter/tree/develop) branch and manually update your wnsm folder. This has a few bugfixes in them regarding problems arising with multiple zaehlpunkte.

**Comment by DarwinsBuddy** [comment]
Duplicate of #227
